spring-boot-admin-server-ui/src/main/frontend/assets/css/base.scss

```diff
@@ -44,6 +44,11 @@ pre.is-breakable {
   white-space: pre-wrap;
 }
 
+.health-details__detail pre {
```

Comment:
this class shouldn't be necessary.

Reply:
Yeah, that's fair. The `monospace` seems a leftover from an earlier iteration of mine. The `<pre/>` font by default seems a little large compared to the surrounding text, hence the `font-size`, but I can just do that in my custom CSS. So, updated code forthcoming.